app/assets/src/stylesheets/02-molecules/forms.scss

```diff
@@ -1,3 +1,28 @@
+// Forms
+//
+// Collects data from users
+//
+// Markup:
+// <form>
+//   <fieldset>
+//     <legend>Submit a Project</legend>
+//     <div class="field">
+//       <label for="project_title">Title</label>
+//       <input type="text">
+//     </div>
+//     <div class="field">
+//       <label for="project_summary">Summary</label>
+//       <input type="text" name="project[summary]" id="project_summary">
+//     <div>
+//   </fieldset>
+//   <div class="actions">
```

Comment:
Is there a way to link back to the molecule we're using here? Or is that not necessary?